src/main/java/com/fnz/TimeTracking/service/AuthService.java

```diff
@@ -2,28 +2,45 @@
 import com.fnz.TimeTracking.model.Utilisateur;
 import lombok.Value;
 import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
 import org.springframework.web.client.RestTemplate;
+import org.springframework.web.multipart.MultipartFile;
 
 @Service
 public class AuthService {
     final private RestTemplate restTemplate;
 
-    //@Value("${python.microservice.url}")
-    private String pythonMicroserviceUrl;
+
+    private String pythonMicroserviceUrl="http://localhost:5000";
 
     AuthService(RestTemplateBuilder restTemplateBuilder) {
         this.restTemplate = restTemplateBuilder.build();
     }
 
-    public Utilisateur validateFace(String image) {
-        String url = pythonMicroserviceUrl + "/validate-face";
-        Utilisateur user = restTemplate.postForObject(url, image, Utilisateur.class);
-        if (user != null && user.isValid()) {
-            return user;
-        } else {
-            return null;
-        }
+    public ResponseEntity<String> validateFace(MultipartFile file) {
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.MULTIPART_FORM_DATA);
+
+        MultiValueMap<String, Object> body = new LinkedMultiValueMap<>();
+        body.add("file", file.getResource());
+
+        HttpEntity<MultiValueMap<String, Object>> requestEntity = new HttpEntity<>(body, headers);
+
+        String url = pythonMicroserviceUrl + "/login";
+        return restTemplate.postForEntity(url, requestEntity, String.class);
+    }
+
+    public Utilisateur loadUserByUsername(String email) {
+        RestTemplate restTemplate = new RestTemplate();
```

Comment:
Don't think we need to ask python for User Details, while spring itself has access to the database.